integrationtest/src/androidTest/java/com/swiftkey/cornedbeef/CoachMarkTestCase.java

```diff
@@ -121,7 +122,28 @@ public void testOnDismissCalled() throws InterruptedException {
         
         verify(mockListener, times(1)).onDismiss();
     }
-    
+
+    /**
+     * Verify that onTimeout is called when coach mark's timeout expired
+     * @throws InterruptedException
+     */
+    public void testOnTimeoutCalled() throws InterruptedException {
+        final long timeout = 1000;
```

Comment:
I've check the possible value down to 10, the test succeed. But I'm afraid that it will become flaky as it have animation. I think around 50ms to 100ms seems okay.